packages/unit-testing/stubs/workbook/Duration.js

```diff
@@ -0,0 +1,48 @@
+define([], function () {
```

Comment:
No action needed.
Missing DimensionSort Objet before this one according to helpdocs.